src/components/Modal.tsx

```
@@ -0,0 +1,91 @@
+import { useState } from "react";
+import { Arrow, Clock, Close, Map_Pin, Plus, Users } from "../assets";
+
+interface ModalProps {
```

ESLint

[error] 'ModalProps' is defined but never used.

(@typescript-eslint/no-unused-vars)